tools/test.py

```diff
@@ -257,6 +260,23 @@ def main():
         args.json_out = args.json_out[:-5]
 
     cfg = mmcv.Config.fromfile(args.config)
+
+    if args.update_config:
+        config_updates = [kv.split('=') for kv in args.update_config]
+        config_updates = [(k.split('.'), v) for (k, v) in config_updates]
+        for k, v in config_updates:
+            cfg_el = cfg
+            stack = []
+            for ki in k:
+                stack.append(cfg_el)
+                cfg_el = cfg_el[ki]
+            cfg_el = v
+            for ki in k[::-1]:
+                a = stack.pop()
+                a[ki] = cfg_el
+                cfg_el = a
+            cfg = cfg_el
```

Comment:
I don't mind this change now. Though would rather replace it with [mmcv's implementation](https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/config.py#L342-L403) as soon as upstream is merged.

Reply:
ok, I see